### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by Devalo** [smpallen99/ex_admin#405]
Looks like a good project...
Would love to try it in a project if its still maintained.

**Comment by Preen** [comment]
They are working on a new version of exadmin!
Its called Talon: https://github.com/talonframework/talon

https://github.com/smpallen99/ex_admin/issues/367

**Comment by sublimecoder** [comment]
@Devalo @Preen Doesn't look like there is a whole lot of work going on there either. It's been 4 months since the last commit on that project, i tried loading up the demo but it seems not quite ready to succeed ExAdmin.

**Comment by Preen** [comment]
lets ask @smpallen99 if he's still actively mainting it or not :)

**Comment by sublimecoder** [comment]
Agreed. If he is not or doesn't have time right now then maybe someone or a few of us can pick up the torch.

**Comment by techgaun** [comment]
yeah I am interested to help this (or talon) project if we can know what's the status and what direction we're heading in. I will try to dig existing documentations and issues to get more information but I am in to contribute from my side to revive this project.

**Comment by Fotografiska** [comment]
we from Flewid would also like to help out as we use this in production envs. @mileandra

**Comment by Preen** [comment]
We from Flewid would also like to help out as we use this in production envs. @mileandra @SmuRf1507

**Comment by sublimecoder** [comment]
Hopefully @smpallen99 hears this and can add a few contributors or switch ownership of the project. if not I will review the two projects talon and ex_admin and select one to fork and create a new project from it. Since there seems to be a fair amount of interest in keeping this alive.

**Comment by MartinElvar** [comment]
@sublimecoder Please keep us in the loop, would be sad to see this go. :grinning:

**Comment by smpallen99** [comment]
@sublimecoder I hear you. I'll spend some time reviewing PRs today.

**Comment by sublimecoder** [comment]
Thanks @smpallen99 would you be open to have a few more maintainers on the project. Maybe taking the load off a bit? Or is Talon still going to be the main focus and we could pitch in there?

**Comment by Preen** [comment]
Yeah, there are lots of issues open that can be faster adressed if we are more people working on it.

**Comment by smpallen99** [comment]
I closed a bunch of PRs last week. Since then there has only been one new PR submitted which I just merged now. 

Everyone is welcome to submit PRs, and I will do my best to get them merged in a timely manner. 

What else does everyone think we can do to encourage people to contribute to the project? 

If someone wants write privileges, let me know and I'll review their past contributions. 

As for Talon, that is where I was hoping to invest my time. However, I have not had any time for open source due to a commercial project that I'm behind schedule on. 

Thanks,
Steve

**Comment by prem-prakash** [comment]
Hello Steve, I would like to have write privileges to help reviewing submitted PRs.

**Comment by MartinElvar** [comment]
@prem-prakash Maybe the solutions could be, to make a fork - Steve seems to be a busy man for the time being :-)

**Comment by sublimecoder** [comment]
Just forked this project and going to start making updates to my own version of ex_admin. Just ran mix format, updated dependencies, and fix deprecation warnings. If anyone wants to submit pull requests, I am more than happy to look at them and get them merged in.

https://github.com/sublimecoder/ex_admin

**Comment by happysalada** [comment]
@sublimecoder just so you know, master is not working properly with phoenix 1.3
If I remember well, the mix install task is not working and updates are not working on the admin interface.

**Comment by fire** [comment]
@sublimecoder Is there a reason why issues on your fork is disabled? I don't want to spam this thread.

**Comment by sublimecoder** [comment]
@fire they are disabled by default. I just enabled them.

**Comment by Preen** [comment]
We have released our beta to the public now:
https://github.com/FLEWID-AB/alkemist
